inference-engine/src/transformations/src/transformations/serialize.cpp

```diff
@@ -198,9 +198,18 @@ std::string get_opset_name(
 
     // return the oldest opset name where node type is present
     for (int idx = 0; idx < opsets.size(); idx++) {
-        if (opsets[idx].get().contains_op_type(n)) {
-            return "opset" + std::to_string(idx + 1);
+        if (!opsets[idx].get().contains_op_type(n)) {
+            continue;
+        }
+
+        if (n->get_type_name() == std::string("ShuffleChannels") &&
```

Comment:
I think it's better to introduce new map (like in translate_type_name()) which will hold operation name and its fixed opset and use it and the beginning of the function to exit early if op is found. It'll be more readable and extensible.

Reply:
Good point, I'll do that